scripts/travis/deploy-linux-build.sh

```diff
@@ -64,19 +64,21 @@ echo "Deploy travis build on the server..."
 # Add server public key to known hosts.
 echo $DEPLOY_SSH_KEY >> $HOME/.ssh/known_hosts
 
-# Send build to server.
+# Remove previous build from server.
 export SSHPASS=$DEPLOY_PASSWORD
+sshpass -e ssh $DEPLOY_USER@$DEPLOY_URL rm -rf $DEPLOY_FOLDER
+
+# Send new build to the server.
 sshpass -e rsync \
     --recursive \
     --archive \
     --compress \
     --stats \
     --no-perms --no-owner --no-group \
     --include="build_report.txt" \
-    --include="sanbox/" \
-    --include="scripts/" \
     --include="prebuilt-linux-deps/" \
-    --delete-after \
+    --include="sandbox/" \
+    --include="scripts/" \ 
```

Comment:
You need the `--exclude` statement as well. Otherwise the whole appleseed directory will be copied over.
Also, in the `--include`s you need the `***`, as you had it a few commits ago.

Reply:
I tested to run it only with include and it was working as expected

Reply:
When there is include rules, rsync will use the first include match to check if a file can be copied. It will be ignored otherwise.

Reply:
*** was required only when i was excluding and deleting. Before that we didn't have *** and it was ok